lib/forklift/box_loader.rb

```diff
@@ -63,13 +63,15 @@ def process_versions(config, versions)
           katello_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-katello-nightly"])
           )
-          katello_box['options'] ||= []
-          katello_box['options'] << " --version #{version}"
+          katello_box['ansible'] ||= {}
+          katello_box['ansible']['variables'] = { 'foreman_version' => version }
 
           capsule_box = Marshal.load(
             Marshal.dump(config['boxes']["#{os}-capsule-nightly"])
           )
+          capsule_box['ansible'] ||= {}
           capsule_box['ansible']['server'] = "#{os}-katello-#{katello_version}"
+          capsule_box['ansible']['variables'] = { 'foreman_version' => version }
```

Comment:
The more I look at it the more I am 50/50 on the dynamic vs. specific boxes. I will work on updating it, but I figured worth a brief discussion/mention.